src/shared/utils/keyEscapeUtils.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule keyEscapeUtils
```

Comment:
Let's capitalize the 'k' to become `KeyEscapeUtils`.  The reason being that we capitalize classes (an object full of methods is basically a class) and reserve lowercase for function names.